runtime-testsuite/test/org/antlr/v4/test/runtime/FileUtils.java

```diff
@@ -60,6 +61,18 @@ public static boolean moveFile(File sourceFir, String destDir, String fileName)
 		return file.renameTo(newFile);
 	}
 
+	public static void replaceInFile(Path sourcePath, String target, String replacement) throws IOException {
+		replaceInFile(sourcePath, sourcePath, target, replacement);
+	}
+
+	public static void replaceInFile(Path sourcePath, Path destPath, String target, String replacement) throws IOException {
+		String content = new String(Files.readAllBytes(sourcePath), StandardCharsets.UTF_8);
```

Comment:
Is it guaranteed that all of the files we are reading will be UTF-8?  Do some of our test files read Japanese or Chinese characters for example?

Reply:
I haven't encountered the problems in out tests.